INSTALL.md

```diff
@@ -1,115 +1,226 @@
+
 Qiita installation
 ==================
 
-Qiita is pip installable, but depends on some non-python packages that must be installed first.
+Qiita is pip installable, but depends on some non-python packages that must be installed first. We strongly recommend using virtual environments; the solution we recommend ot manage them is [miniconda](http://conda.pydata.org/miniconda.html), a lightweight version of the virtual environment, python distribution, and package manager anaconda.
```

Comment:
I'm not sure if miniconda is the version we recommend ... I would say that is the version we will use in this manual. Note that we use virtualenv in all our internal installs.